ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/validation/ModelConformanceValidatorSpec.scala

```diff
@@ -375,6 +375,20 @@ class ModelConformanceValidatorSpec
         .build()
       step shouldBe StepStop(expectedEntry)
     }
+
+    "accept a transaction in case an input contract ID is non-existent" in {
```

Comment:
Can you explain somewhere _why_ a contract ID might disappear? If it's related to committer pruning, let's write that down.

Reply:
Changed the name of the test case to include that.